phpBB/phpbb/avatar/driver/upload.php

```diff
@@ -13,15 +13,17 @@
 
 namespace phpbb\avatar\driver;
 
+use \phpbb\storage\exception\exception as storage_exception;
```

Comment:
What I meant is that it could be a good idea to rename \phpbb\storage\exception\exception to \phpbb\storage\exception\storage_exception

What do you think?